React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -360,12 +362,62 @@ - (void)updateProps:(Props::Shared const &)props oldProps:(Props::Shared const &
   if (oldViewProps.testId != newViewProps.testId) {
     self.accessibilityIdentifier = RCTNSStringFromString(newViewProps.testId);
   }
-
+  
+  BOOL accessibilityLiveRegionEnabled = newViewProps.accessibilityLiveRegion != AccessibilityLiveRegion::None;
+  BOOL childAccessibilityLiveRegionEnabled = ![self.accessibilityLiveRegion isEqual:@"none"];
+  BOOL isReactRootView = RCTIsReactRootView(self.reactTag);
+  if ((accessibilityLiveRegionEnabled || childAccessibilityLiveRegionEnabled) && !isReactRootView) {
```

Comment:
should trigger reset of child instance variable when changes from polite to none, as children needs to change also